src/services/supabase.ts

```diff
@@ -25,6 +25,9 @@ export const CONNECTOR_IMAGE = `image:connector_logo_url->>en-US`;
 
 export const QUERY_PARAM_CONNECTOR_TITLE = `connector_title->>en-US`;
 
+export const SHARDS_DISABLE = `spec->shards->disable`;
+export const TASK_STATS = `task_stats:flow_document->>taskStats::json`;
```

Comment:
Ugh... I don't know why I made this so complex... I think we can just use this since it is already stored as `json`

`task_stats:flow_document->taskStats`